tests/dat/actions/base64Web.js

```diff
@@ -0,0 +1,9 @@
+function main() {
+	  return {
+		  headers: {
+			  "content-type": "application/json"
+		  },
+		  statusCode: 200,
+		  body: new Buffer(JSON.stringify({'status': 'success'})).toString('base64')
+	  }
+}
```

Comment:
@markusthoemmes Currently (and historically), the "tests" directory and its subdirs are listed in the "exclusions" section of the default scancode.cfg file.  Messaged Rodric that I would be happy to remove (and want to remove ALL exclusions), but have been waiting for dozens of PRs to merge to allow even considering this.  I will work to see what impact removing the "tests" folder exclusion and see if I can submit a PR.